source/tests/system/nidmm_driver_api_tests.cpp

```diff
@@ -269,6 +321,72 @@ TEST_F(NiDmmDriverApiTest, ConfigureCurrentSourse_CompletesSuccessfully)
     expect_api_success(response.status());
 }
 
+TEST_F(NiDmmDriverApiTest, ExportConfiguredCurrentSource_ResetAndImportConfiguration_CurrentSourceConfigurationIsImported)
+{
+    float expected_value = 0.0001;
+    configure_current_source(expected_value);
+    
```

Comment:
The empty lines seems to have got messed up for this method.

Reply:
fixed